core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
No, I'm afraid they are not. Please pay attention to indentation, and the position of the style tag in the file. See the [previous comment](https://github.com/oppia/oppia/pull/3193#discussion_r105592255) I made, and compare the file you are editing to the file mentioned in that comment.